### PR TITLE
Nicer Try commit messages

### LINKED ISSUE #1990: Cleaner commit messages when pushing to try
Each commit in a try push should have:
* the original commit message
* the phabricator revision url

Bonus: indent try_task_config.json

### PR DESCRIPTION
**Pull request by La0**
As requested on IRC, i modified the messages for our Try pushes commits by reusing the original message from developer. In most cases the Phabricator revision URL is already present in that message.

Fixes #1990 .

### CODE REVIEW

File: src/pulselistener/pulselistener/mercurial.py [hunk 1]
```diff
@@ -115,12 +115,30 @@ def clean(self):
...
+        )
+        commits = {
+            diff['phid']: diff['attachments']['commits'].get('commits', [])
+            for diff in diffs
+        }
+
         # Apply the patches and commit them one by one
         for diff_phid, patch in patches:
-            logger.info('Applying patch', phid=diff_phid)
+            commit = commits.get(diff_phid)
```
Comment (marco-c) [inline src/pulselistener/pulselistener/mercurial.py]: There are cases in which commits doesn't have a diff_phid?

File: src/pulselistener/pulselistener/mercurial.py [hunk 2]
```diff
@@ -115,12 +115,30 @@ def clean(self):
...
+            if commit:
+                message = '{}\nDiff: {}'.format(commit[0]['message'], diff_phid)
```
Comment (marco-c) [inline src/pulselistener/pulselistener/mercurial.py]: The Diff ID is not already in the commit message?

Comment (marco-c) [reply src/pulselistener/pulselistener/mercurial.py]: Maybe you can keep the same convention as commits landed via Phab, which add `Differential Revision: https://phabricator.services.mozilla.com/DXXX` at the end (e.g. https://hg.mozilla.org/mozilla-central/rev/788b5dbea4e8).

File: src/pulselistener/pulselistener/mercurial.py (hunk 1 above)
Comment (La0) [reply src/pulselistener/pulselistener/mercurial.py]: Someone could post a patch on phabricator without using a commit.

File: src/pulselistener/pulselistener/mercurial.py (hunk 2 above)
Comment (La0) [reply src/pulselistener/pulselistener/mercurial.py]: Only the revision `D1234` is mentionned, i'll use the format you mentionned for PHID, but there are no url pointing to a `PHID-DIFF`

**Pull request merged**